recipes/plotnn/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
It looks like the license file is included in the source distribution so you don't need to include another copy here

Reply:
Thank you, I removed the redundant license file.